api/routes/tasks.py

```diff
@@ -19,56 +19,108 @@ def get_tasks(user_id):
         [
             {
                 "name": t.title,
-                "id": t.id,
+                "id": t.task_id,
                 "description": t.description,
-                "complete": t.complete is not None,
-                "duration": t.duration,
+                "complete": t.completed is not None,
+                "duration": t.duration_minutes,
             }
             for t in tasks
         ]
     )
 
 
+@app.route("/api/task/create", methods=["POST"])
+@with_user_id
+def create_task(user_id):
+    body = request.get_json()
+
+    if "name" not in body:
+        abort(401)
```

Comment:
Is 401 an appropriate code in this case? It's not that the name in the request is unauthorized, it's that it wasn't present in the request, so the request is bad